使用时不定时会发生闪退

### 复现步骤 / Step to reproduce

正常使用时随机出现

### 预期行为 / Expected behaviour

正常响应

### 实际行为 / Actual Behaviour

闪退

### 应用日志 / App logs

闪退时系统弹窗提示：因应用自身空指针异常，造成闪退
问题Trace：
`java.lang.RuntimeException: Unable to destroy activity {moe.tarsin.ehviewer/eu.kanade.tachiyomi.ui.reader.ReaderActivity}: java.lang.NullPointerException
	at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:5774)
	at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:5806)
	at android.app.servertransaction.DestroyActivityItem.execute(DestroyActivityItem.java:47)
	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:180)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:98)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2545)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:222)
	at android.os.Looper.loop(Looper.java:314)
	at android.app.ActivityThread.main(ActivityThread.java:8530)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:561)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1010)
Caused by: java.lang.NullPointerException
	at Lz0.f(Unknown Source:29)
	at lc0.e(Unknown Source:0)
	at lc0.g(Unknown Source:97)
	at nG.v(Unknown Source:3)
	at eu.kanade.tachiyomi.ui.reader.ReaderActivity.onDestroy(Unknown Source:22)
	at android.app.Activity.performDestroy(Activity.java:9165)
	at android.app.Instrumentation.callActivityOnDestroy(Instrumentation.java:1503)
	at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:5761)
	... 13 more
`

### 备注 / Addition details

_No response_

### 浏览站点 / Browsing site

ExHentai

### EhViewer 版本号 / EhViewer version code

1.8.10.2-gms

### EhViewer CI 版本 / EhViewer CI Version

10c7019

### Android 系统版本 / Android version

MIUI 14.0.3 14.0.3.0(UMCCNXM) ,Android 14

### 设备型号 / Device model

Xiaomi13

### SoC 型号 / Soc model

Snapdragon 8gen2

### 自查步骤 / Verify steps

- [ ] 如果您有足够的时间和能力，并愿意为此提交 PR ，请勾上此复选框 / Pull request is welcome. Check this if you want to start a pull request
- [X] 您已搜索过 [Issue Tracker](https://github.com/FooIbar/EhViewer/issues)，没有找到类似内容 / I have searched on [Issue Tracker](https://github.com/FooIbar/EhViewer/issues), No duplicate or related open issue has been found
- [X] 您确保这个 Issue 只提及一个问题。如果您有多个问题报告，烦请发起多个 Issue / Ensure there is only one bug report in this issue. Please make mutiply issue for mutiply bugs
- [X] 您确保已使用最新 [CI 版本](https://github.com/FooIbar/EhViewer/actions)测试，并且该问题在最新 CI 版本中并未解决 / This bug have not solved in latest [CI version](https://github.com/FooIbar/EhViewer/actions)

> 你这日志不全，使用应用的导出日志功能导出
`TIME=2023-10-28-10-54-38-573

======== PackageInfo ========
PackageName=moe.tarsin.ehviewer
VersionName=1.8.10.2-gms
VersionCode=180044
CommitSha=10c7019
BuildTime=2023/10/27 11:49

======== DeviceInfo ========
BOARD=kalama
BOOTLOADER=unknown
BRAND=Xiaomi
CPU_ABI=armeabi-v7a
CPU_ABI2=armeabi
DEVICE=fuxi
DISPLAY=UKQ1.230705.002 release-keys
FINGERPRINT=Xiaomi/fuxi/fuxi:14/UKQ1.230705.002/V14.0.3.0.UMCCNXM:user/release-keys
HARDWARE=qcom
HOST=pangu-build-component-system-415801-hml0j-f11vp-0qngh
ID=UKQ1.230705.002
IS_DEBUGGABLE=false
IS_EMULATOR=false
IS_MIUI=true
MANUFACTURER=Xiaomi
MODEL=2211133C
ODM_SKU=fuxi
PERMISSIONS_REVIEW_REQUIRED=true
PRODUCT=fuxi
RADIO=unknown
SERIAL=unknown
SKU=fuxi
SOC_MANUFACTURER=QTI
SOC_MODEL=SM8550
SUPPORTED_32_BIT_ABIS=armeabi-v7a, armeabi
SUPPORTED_64_BIT_ABIS=arm64-v8a
SUPPORTED_ABIS=arm64-v8a, armeabi-v7a, armeabi
TAGS=release-keys
TIME=1695654851000
TYPE=user
UNKNOWN=unknown
USER=builder
ACTIVE_CODENAMES=
BASE_OS=
CODENAME=REL
INCREMENTAL=V14.0.3.0.UMCCNXM
KNOWN_CODENAMES={HoneycombMr1, HoneycombMr2, Lollipop, Kitkat, Tiramisu, Gingerbread, Cupcake, IceCreamSandwichMr1, JellyBean, IceCreamSandwich, LollipopMr1, M, N, O, P, Q, R, S, Sv2, Base, NMr1, OMr1, JellyBeanMr1, JellyBeanMr2, Donut, Froyo, GingerbreadMr1, EclairMr1, UpsideDownCake, Honeycomb, Eclair01, KitkatWatch, Base11, Eclair}
MEDIA_PERFORMANCE_CLASS=33
PREVIEW_SDK_FINGERPRINT=REL
PREVIEW_SDK_INT=0
RELEASE=14
RELEASE_OR_CODENAME=14
RELEASE_OR_PREVIEW_DISPLAY=14
SDK=34
SDK_INT=34
SECURITY_PATCH=2023-09-05
MEMORY=69.8 MiB
MEMORY_NATIVE=60.1 MiB
MEMORY_MAX=256.0 MiB
MEMORY_TOTAL=10.9 GiB

======== CrashInfo ========
java.lang.RuntimeException: Unable to destroy activity {moe.tarsin.ehviewer/eu.kanade.tachiyomi.ui.reader.ReaderActivity}: java.lang.NullPointerException
	at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:5774)
	at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:5806)
	at android.app.servertransaction.DestroyActivityItem.execute(DestroyActivityItem.java:47)
	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:180)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:98)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2545)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:222)
	at android.os.Looper.loop(Looper.java:314)
	at android.app.ActivityThread.main(ActivityThread.java:8530)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:561)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1010)
Caused by: java.lang.NullPointerException
	at Lz0.f(Unknown Source:29)
	at lc0.e(Unknown Source:0)
	at lc0.g(Unknown Source:97)
	at nG.v(Unknown Source:3)
	at eu.kanade.tachiyomi.ui.reader.ReaderActivity.onDestroy(Unknown Source:22)
	at android.app.Activity.performDestroy(Activity.java:9165)
	at android.app.Instrumentation.callActivityOnDestroy(Instrumentation.java:1503)
	at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:5761)
	... 13 more
java.lang.NullPointerException
	at Lz0.f(Unknown Source:29)
	at lc0.e(Unknown Source:0)
	at lc0.g(Unknown Source:97)
	at nG.v(Unknown Source:3)
	at eu.kanade.tachiyomi.ui.reader.ReaderActivity.onDestroy(Unknown Source:22)
	at android.app.Activity.performDestroy(Activity.java:9165)
	at android.app.Instrumentation.callActivityOnDestroy(Instrumentation.java:1503)
	at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:5761)
	at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:5806)
	at android.app.servertransaction.DestroyActivityItem.execute(DestroyActivityItem.java:47)
	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:180)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:98)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2545)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:222)
	at android.os.Looper.loop(Looper.java:314)
	at android.app.ActivityThread.main(ActivityThread.java:8530)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:561)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1010)
`

--- a/app/src/main/kotlin/com/hippo/ehviewer/image/Image.kt
+++ b/app/src/main/kotlin/com/hippo/ehviewer/image/Image.kt
@@ -56,8 +56,7 @@ class Image private constructor(drawable: Drawable, private val src: AutoCloseab
     var mObtainedDrawable: Drawable? = drawable
         private set
 
-    val size: Int
-        get() = mObtainedDrawable!!.run { intrinsicHeight * intrinsicWidth * 4 * if (this is Animatable) 4 else 1 }
+    val size = drawable.run { intrinsicHeight * intrinsicWidth * 4 * if (this is Animatable) 4 else 1 }
 
     @Synchronized
     fun recycle() {

